src/doc/topics/code-structure.md

```diff
@@ -0,0 +1,114 @@
+# Code and Project Structure
+
+## Code Structure
+
+We want to keep a standard code structure that we base on the original Reusable Chart API. The structure of a chart would look more or less like this:
```

Comment:
"We require contributors to preserve both the structure and the code standards based on the original Reusable Chart API"